packages/core/security/core-security-server/src/contracts.ts

```diff
@@ -21,6 +22,11 @@ export interface SecurityServiceSetup {
    * @remark this should **exclusively** be used by the security plugin.
    */
   registerSecurityDelegate(api: CoreSecurityDelegateContract): void;
+
+  /**
+   * The {{@link CoreFipsService | FIPS service}}
```

Comment:
NIT: I don't think doubling `{{` `}}` even works for tsdoc?

Reply:
Fixed in https://github.com/elastic/kibana/pull/181187/commits/7b7ed010889b5072d430622be372d4af9bfee487